spec/functions/postfix_table_type_extensions_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'postfix::table_type_extension' do
+  context 'random string' do
+    it { is_expected.to run.with_params('foo').and_return('db') }
+  end
+
+  context 'with lmdb' do
+    it { is_expected.to run.with_params('lmdb').and_return('lmdb') }
+  end
+
+  context 'with pcrm' do
```

Comment:
Not sure if this is a typo. Otherwise LGTM.

```suggestion
  context 'with pcre' do
```